src/libs/antares/study/include/antares/study/parts/short-term-storage/cluster.h

```diff
@@ -47,5 +47,6 @@ class STStorageCluster
 
     std::shared_ptr<Series> series = std::make_shared<Series>();
     mutable Properties properties;
+    void SynchronizePenaltyOnVariation() const;
```

Comment:
You probably don't need this synchronization mechanism.

However, you can move `struct PenaltyCostOnVariation` to `class Properties`, and remove anything related to penalties in `class Series`.